cmd/cluster-display/main.go

```diff
@@ -14,6 +14,7 @@ import (
 	"time"
 
 	"github.com/sirupsen/logrus"
+	"golang.org/x/time/rate"
```

Comment:
Looks like we also need to update `go.mod` ?

Reply:
Yes, updated. Thanks!